extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayCreator.java

```diff
@@ -54,6 +54,7 @@ public Flyway createFlyway(DataSource dataSource) {
         if (flywayRuntimeConfig.repeatableSqlMigrationPrefix.isPresent()) {
             configure.repeatableSqlMigrationPrefix(flywayRuntimeConfig.repeatableSqlMigrationPrefix.get());
         }
+        configure.batch(flywayRuntimeConfig.batch);
```

Comment:
If not configured, what's the default Flyway behaviour? Asking because we might want to preserve it with a default value in ConfigItem.

Reply:
Default is `false`, see https://flywaydb.org/documentation/configuration/parameters/batch